sql/moz-fx-data-shared-prod/udf_js/decode_uri_attribution/metadata.yaml

```
@@ -0,0 +1,5 @@
+description: URL decodes the raw firefox_installer.install.attribution string
+  to a STRUCT.  The fields campaign, content, dlsource, dltoken, experiment,
+  medium, source, ua, variation the string are extracted.  If any value is
+  (not+set) is is converted to (not set).
```

nit: maybe add Why `(not set)` is preferable to `(not+set)` in the description? 
```suggestion
  medium, source, ua, variation the string are extracted.  If any value is
  (not+set) it is converted to (not set) to match the text from GA when the fields are not set.
```